docs/docusaurus.config.js

```diff
@@ -77,8 +98,8 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
-      // image is expected to be a "social card".  Logo for now.
-      image: "img/gl-icon400.png",
+      // Replace with your project's social card
+      image: "img/docusaurus-social-card.jpg",
```

Comment:
I think this is accidentally reverting us to use the Docusaurus icon instead of the Groundlight Icon

Reply:
thanks for catching this Tyler!

Reply:
Another merge mistake?  We shouldn't use the docusaurus card.